UX module not exported correctly

**Describe the bug**
The action import in the documentation is incorrect: https://github.com/oclif/core/blob/main/src/ux/README.md

It suggests that it should be imported like:
```
import { action } from "@oclif/core/ux"
```

However, it has to be imported like:

```
import { ux } from "@oclif/core/ux"
const { action } = ux
```

**To Reproduce**

Minimal repro provided here: https://github.com/s-palmer/oclif-ux-package-repro

Steps to reproduce the behavior:
1. Clone the above repo
2. `npm install`
3. `npm exec tsc`
4. See error below:

```
src/commands/hello.ts:2:10 - error TS2305: Module '"@oclif/core/ux"' has no exported member 'action'.
```

ux module imported as per documentation: https://github.com/oclif/core/blob/main/src/ux/README.md

**Expected behavior**
The UX module should be able to be imported as per the documentation

**Screenshots**
If applicable, add screenshots to help explain your problem.

![image](https://github.com/user-attachments/assets/6a24ba3a-fb12-435a-990d-95e0dd307dcd)

The suggested import doesn't work. The ux module can be accessed as per the commented out code below:

```
import { action } from "@oclif/core/ux"
// import ux from "@oclif/core/ux"
// const { action } = ux.ux
```

**Environment (please complete the following information):**
 - OS & version: MacOS 14.7.1
 - Shell/terminal & version: zsh 5.9 (x86_64-apple-darwin23.0)

--- a/src/ux/index.ts
+++ b/src/ux/index.ts
@@ -21,8 +21,10 @@ const ACTION_TYPE =
     'spinner') ||
   'simple'
 
+export const action = ACTION_TYPE === 'spinner' ? new Spinner() : new Simple()
+
 export const ux = {
-  action: ACTION_TYPE === 'spinner' ? new Spinner() : new Simple(),
+  action,
   /**
    * Add color to text.
    * @param color color to use. Can be hex code (e.g. `#ff0000`), rgb (e.g. `rgb(255, 255, 255)`) or a standard ansi color (e.g. `red`)

